package.json

```diff
@@ -1,7 +1,8 @@
 {
   "name": "react-boilerplate",
   "version": "3.5.0",
-  "description": "A highly scalable, offline-first foundation with the best DX and a focus on performance and best practices",
+  "description":
+    "A highly scalable, offline-first foundation with the best DX and a focus on performance and best practices",
```

Comment:
A lot of the prettier changes below are good IMO (like lines 67-69 becoming only one line and the likes).

However, could we remove the ones based on the character limit which result in key and value being on different lines? They look weird and don't help much since the value itself is still above the character limit.